sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/UnsupportedOperationsSuite.scala

```diff
@@ -414,209 +412,135 @@ class UnsupportedOperationsSuite extends SparkFunSuite {
     batchStreamSupported = false,
     streamBatchSupported = false)
 
-  // Left outer joins: *-stream not allowed
+  // Left outer, left semi, left anti join: *-stream not allowed
+  Seq((LeftOuter, "LeftOuter join"), (LeftSemi, "LeftSemi join"), (LeftAnti, "Left anti join"))
```

Comment:
super nit, let's keep naming style here? `LeftOut join` ... `LeftAnti join` or `left outer join` ... `left anti join`.

Reply:
@xuanyuanking - wondering what's the naming style we are trying to following here? TBH I would be surprised that someone fails to understand comment of `left outer`, if I omit `join` here. Similar comments happened several times in other files e.g. in [StreamingSymmetricHashJoinExec](https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/StreamingSymmetricHashJoinExec.scala#L325) . But anyway I feel it's too nit, if you think it's worth the change, I can do it as well.

Reply:
I mean at least be consistent for the `name` param here:
`LeftOuter join` `LeftSemi join` and `LeftAnti join`
or
`left outer join` `left simi join` and `left anti join`

Yes, it's too nit and not worth a single commit if there's no further comment.

Reply:
This is nit but super easy to fix, so probably better to fix now once we find it. It'd be appreciated if you can fix this - won't drag the review process as @xuanyuanking already provided LGTM and this is probably going to be a final review comment.

Reply:
@c21 
No his suggestion is not about the comment. It's in `name` part, `Left anti join` -> `LeftAnti join`, as for other joins you don't add a space.

Reply:
Sorry that I misunderstood in the first place. Changed to `LeftAnti join` and https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/UnsupportedOperationChecker.scala#L307 as well.